Week01/Problem01/gwang_01.py

```diff
@@ -0,0 +1,6 @@
+#gwang_01.py
+j = 0
+for i in range (1000):
+    if (i % 3) == 0 or (i%5) == 0: j+=i
+j
```

Comment:
This won't actually print anything, unless you're in Spyder or some other IDE which behaves like Matlab...